src/app/devices/module/moduleIndentityDetail/sagas/deleteModuleIdentitySaga.spec.ts

```diff
@@ -0,0 +1,82 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import 'jest';
+// tslint:disable-next-line: no-implicit-dependencies
+import { SagaIteratorClone, cloneableGenerator } from '@redux-saga/testing-utils';
+import { call, put } from 'redux-saga/effects';
+import { deleteModuleIdentitySaga } from './deleteModuleIdentitySaga';
+import { deleteModuleIdentityAction } from '../actions';
+import * as ModuleService from '../../../../api/services/moduleService';
+import { ResourceKeys } from '../../../../../localization/resourceKeys';
+import { NotificationType } from '../../../../api/models/notification';
+import { ModuleIdentity } from '../../../../api/models/moduleIdentity';
+import { raiseNotificationToast } from '../../../../notifications/components/notificationToast';
+
+describe('moduleIdentitySaga', () => {
```

Comment:
would be deleteModuleIdentitySaga (might be copied from the old file which was wrong...)